docs/docs/1.1.x/installation/amazonlinux.md

```diff
@@ -16,10 +16,12 @@ Finally you can follow the [Quickstart](#_4-quickstart) to take it from here and
 
 Run the following script to automatically detect the operating system and download Kuma:
 
-```sh
+<div class="language-sh">
+<pre><code>
 yum install -y tar gzip
-curl -L https://kuma.io/installer.sh | sh -
-```
+curl -L https://kuma.io/installer.sh | VERSION={{ $page.latestVersion }} bash -
```

Comment:
**Issue**: There are several changes that switch the shell from `sh` to `bash`. I don’t see a reason to do this. Please change those instances to `sh` (only in those cases that haven’t already been using `bash`).